en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -24,27 +24,23 @@ doi: XX.XXXXX/phen0000
 
 ## Introduction
 
-We no longer require students to memorize tables of historical events and their respective years of occurrence through the practice of rote learning, as was done in mid-nineteenth century America (Rosenberg). Setting aside the broader arguments for and against teaching chronology in the classroom (Hodkinson and Smith), this lesson is written from the position that it is useful to “help students develop a rich body of knowledge in our content areas” (Lang) and that games are well situated to serve in this pursuit. 
+We no longer require students to memorize tables of historical events and their respective years of occurrence through the practice of rote learning, as was done in mid-nineteenth century America.[^1] Setting aside the broader arguments for and against teaching chronology in the classroom,[^2] this lesson assumes that it is useful to 'help students develop a rich body of knowledge in our content areas'[^3] and that games are well situated to serve in this pursuit. 
```

Comment:
As you can see already from this first line, I've changed all the citations to endnotes, as all our lessons follow the Chicago Manual of Style, 17th edition.